lib/Jsrt/Core/JsrtContextCore.h

```diff
@@ -163,7 +163,6 @@ class ChakraCoreHostScriptContext sealed : public HostScriptContext
 
     HRESULT EnqueuePromiseTask(Js::Var taskVar) override
     {
-        AssertMsg(false, "jsrt should have set the promise callback");
```

Comment:
I feel like this should be ok as well, it only hits in debug builds when the embedder has not added a promise callback, but tries to use Promises in their code?